docusaurus.config.js

```diff
@@ -218,7 +218,7 @@ module.exports = {
       },
     ],
     [
-      // Zeebe REST API docs generation (removed at v8.7)
+      // Zeebe REST API docs generation
```

Comment:
Zeebe REST API and Tasklist GraphQL are both in a deprecated section. Do we want to make a comment here about this?